src/AppInstallerCLICore/Commands/RepairCommand.cpp

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "RepairCommand.h"
+#include "Workflows/RepairFlow.h"
+#include "Workflows/CompletionFlow.h"
+#include "Workflows/InstallFlow.h"
+
+namespace AppInstaller::CLI
+{
+    using namespace AppInstaller::CLI::Execution;
+    using namespace AppInstaller::CLI::Workflow;
+
+    std::vector<Argument> RepairCommand::GetArguments() const
+    {
+        return {
+            Argument::ForType(Args::Type::Query),                            // -q
+            Argument::ForType(Args::Type::Manifest),                         // -m
+            Argument::ForType(Args::Type::Id),                               // -id
+            Argument::ForType(Args::Type::Name),                             // -n
+            Argument::ForType(Args::Type::Moniker),                          // -mn
+            Argument::ForType(Args::Type::Version),                          // -v
+            Argument::ForType(Args::Type::InstallArchitecture),              // -arch
+            Argument::ForType(Args::Type::Source),                           // -s
+            Argument::ForType(Args::Type::Interactive),                      // -i
+            Argument::ForType(Args::Type::Silent),                           // -h
+            Argument::ForType(Args::Type::Log),                              // -o
+            Argument::ForType(Args::Type::IgnoreLocalArchiveMalwareScan),    // -ignore-local-archive-malware-scan
+            Argument::ForType(Args::Type::AcceptSourceAgreements),           // -accept-source-agreements
+            Argument::ForType(Args::Type::Locale),

```

Comment:
Also add CustomHeader, AuthenticationMode, AuthenticationAccount, InstallScope, ProductCode, Force, HashOverride, Exact, Channel(though not supported yet, just to be consistent with other commands)